.github/workflows/release-build.yml

```diff
@@ -12,8 +12,11 @@ env:
 
 jobs:
   release-build:
-    runs-on: ubuntu-latest
-
+    runs-on: oxide-colo-builder-hubris
```

Comment:
I assume this change is intentional?

Reply:
Correct. This was noted in the commit text which github does a great job of making it hard to find
```
commit cc10be0a42aaee8ea3935d17ddf2aac8f3ad6923 (HEAD -> xtask, origin/xtask)
Author: Laura Abbott <laura@oxide.computer>
Date:   Fri Jan 31 15:08:30 2025 -0500

    Update github actions
    
    - Build on the Oxide Colo runner
    - Package and attest
```